test/firk/expanded/nlls_tests.jl

```diff
@@ -0,0 +1,208 @@
+# Overconstrained and Underconstrained are taking too much time, skip them for now
+#=
```

Comment:
should this just be a different test group?

Reply:
Yeah, but the CI takes a long time to test them so just commented them out for now.

Reply:
An ugly way to do this is https://github.com/LuxDL/LuxLib.jl/blob/main/test/common_ops/dense_tests.jl. Create a module with the test and run it in separate groups, that way you get parallelism.

Also if the tests are too heavy we can move these to the buildkite cpus.

Reply:
Let's try putting them into a different test group

Reply:
The nlls tests for Lobatto and Radau methods take so much time😅: https://github.com/SciML/BoundaryValueDiffEq.jl/actions/runs/10447244427/job/28925868285?pr=178

Are there some issues with the methods or something else?

Reply:
what's it spending all of that time doing?

Reply:
Just to confirm this is not some issue with parallel testing, let's disable parallel testing for this PR

Reply:
Just figured out the problem, the last commit has changed the number of expanded solvers for nlls tests to 4*4=16, and the CI says nlls tests cost approximately 37minutes  `2270.1 secs (36.1% compile, 18.6% GC), 2.57 B allocs (314.128 GB)`😅. The previous nlls testing cases test all of the 68 FIRK methods, which would need a looooong time to finish😅.